docs/rest_api_ep.rst

```diff
@@ -0,0 +1,274 @@
+REST API Dynamic Endpoint Attach Detach
+=======================================
+
+This section will go over the specifics of the REST APIs supported by F5 ACI ServiceCenter application for dynamic endpoint attach detach feature.
+
+The REST calls can be made to the following APIC endpoint
+
+.. code-block:: json
+
+   https://<APIC-URL>/appcenter/F5Networks/F5ACIServiceCenter/<REST-API>
+
+   Example:
+
+   https://10.107.0.24/appcenter/F5Networks/F5ACIServiceCenter/getbigiplist.json
+
+`Download the API POSTMAN collection <https://github.com/F5Networks/f5-aci-servicecenter/tree/master/api_collection>`_.
+
+Request Headers
+---------------
+
+For all the F5 ACI ServiceCenter API calls following request header parameters are required:
+
+Header: DevCookie
+`````````````````
+Can be retrieved as follows:
+
+1. POST a request to the following API endpoint
+
+.. code-block:: json 
+
+   https://<APIC-URL>/api/aaaLogin.xml
+
+2. Body for the POST request
+
+.. code-block:: json
+
+   <aaaUser name="apic-username" pwd="apic-password"/>
+
+3. From the result of the POST, save the token returned
+
+4. In the “Headers” section of any further REST API requests, add a key-value pair. Key name is “DevCookie”, and its value should be the token obtained in the previous step
```

Comment:
4. In the **Headers** section of any further REST API requests, add a key-value pair. Key name is **DevCookie**, and its value should be the token obtained in the previous step

Reply:
Resolved in docs/rest_api.rst (docs/rest_api_ep.rst  is an outdated filename)